lib/spack/docs/binary_caches.rst

```diff
@@ -57,6 +57,26 @@ Build caches are installed via:
 
    $ spack buildcache install
 
+Note that the above command is intended to install a particular package to a
+build cache you have created, and not to install a package from a build cache.
+For the latter, once a mirror is added, by default when you do ``spack install`` the ``--use-cache``
+flag is set, and you will install a package from a build cache if it is available.
+If you want to always use the cache, you can do:
+
+.. code-block:: console
+
+   $ spack install --cache-only <package>
+
+For example, to combine all of the commands above to add the E4S build cache
+and then install from it exclusively, you would do:
+
+.. code-block:: console
+
+    $ spack mirror add E4S https://cache.e4s.io
+    $ spack buildcache keys -it
```

Comment:
Since the abbreviated version of the arguments aren't described in this area, I suggest using the full options:

```suggestion
    $ spack buildcache keys --install --trust
```

You might also want to explain why you  added `-t` (or `--trust`).

Reply:
I don't actually know haha, the example in the current docs just showed using it. Do you know why would I want to add it?

Reply:
Coming from container land when I see -it I usually think "interactive terminal" :laughing:

Reply:
okay gave it a shot! I might need some help in flushing out what is really going on when we add trust.